src/components/visualizations/chart/highcharts/plugins/dualAxesLabelFormatter.ts

```diff
@@ -4,6 +4,41 @@ import head = require('lodash/head');
 import last = require('lodash/last');
 import { formatAsPercent, isInPercent } from '../customConfiguration';
 
+const DEFAULT_LIMIT_LENGTH = 4; // 0.00
+const DEFAULT_PADDING = 2;
+
+/**
+ * Get limit length to cut off value less than one
+ * @param value
+ */
+function getLimitLength(value: number): number {
+    const valueStr = value.toString();
+    const isFloat = valueStr.indexOf('.') > 0;
+    const length = isFloat ? valueStr.length : valueStr.length + 1;
+    return length + DEFAULT_PADDING;
+}
+
+function isValueInShallowRange(min: number, max: number): boolean {
+    return min !== max && Math.abs(max - min) < 10;
```

Comment:
What is number 10? Shouldn't this be in constant?

Reply:
Yes, it should :)